### DISCUSSION TITLE
run pubpub locally

### DISCUSSION THREAD

**Discussion by tiwarishankar** [knowledgefutures/pubpub#3121] [General]
How i can run Pubpub locally or still it is not up for running locally?
i tried to run it locally by going through readme.md but wasn't able to run !!

**Comment by eastofesten** [comment]
Hello! PubPub Legacy is not up for running locally, but we have been hard at work on [PubPub Platform](https://github.com/pubpub/platform), a tool that allows communities of all kinds to build knowledge applications tailored to their needs. We invite you to install and explore the alpha version of PubPub Platform, and we welcome your feedback.

Learn More: [Documentation](https://knowledgefutures.helpjuice.com/) | [PubPub Platform](https://knowledgefutures.org/pubpub) | [Knowledge Futures](https://www.knowledgefutures.org/) | [Newsletter](https://pubpub.us5.list-manage.com/subscribe?u=9b9b78707f3dd62d0d47ec03d&id=be26e45660) | [Roadmap](https://github.com/orgs/pubpub/projects/46/views/1)
